api/routes/SignLog.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express')
+const router = express.Router()
+const SignLog = require('../models/SignLog')
+const { OK, BAD_REQUEST } = require('../constants').STATUS_CODES
+
+router.post('/addSignLogs', (req, res) => {
+  const newSign = new SignLog({
+    signTitle: req.body.signTitle,
+    firstName: req.body.firstName,
+    email: req.body.email,
+    timeOfPosting: req.body.timeOfPosting
+  })
+
+  newSign.save(function (sign) {
```

Comment:
can you rename this parameter to:
```suggestion
  newSign.save(function (error) {
```
that way `if(error)` makes more sense